src/cart/crt_hg.c

```diff
@@ -42,6 +42,12 @@ struct crt_na_dict crt_na_dict[] = {
 		.nad_str	= "ofi+psm2",
 		.nad_contig_eps	= false,
 		.nad_port_bind  = false,
+	}, {
+		.nad_type	= CRT_NA_OFI_OPX,
+		.nad_str	= "ofi+opx",
+		.nad_alt_str	= "opx",
+		.nad_contig_eps	= true,
```

Comment:
contig_eps should be really set to false here, since we cannot yet generate uri of 'endpoint + 1' from uri of the 'endpoint'. And as a result code in crt_get_info_string() should be moved outside of crt_provider_is_contig_ep() clause. I've done those changes to the master version